daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1318,7 +1318,18 @@ private[lf] object SBuiltin {
       }
   }
 
-  /** $pass :: Int64 -> Token -> Timestamp */
+  /** $pure :: a -> Update a */
```

Comment:
There is no change to the code here. This is just a comment change.
But I think I have made it more confusing. I will change back

Reply:
You added `SBSPure` didn’t you?

Reply:
Yeah I did. I wrote about this in the (perhaps overly long) PR description.

Reply:
ah I see. Seems like my point still stands though. You’re adding an extra wrapper which seems like it might have a performance impact and it seems unrelated to this PR.

Reply:
yes. it is slightly unrelated. if people feel strongly it could be moved to a independent  PR or killed entirely.

Regarding performance. I would argue it is likely to be quicker. The original code constructed an application node instead of a a builtin. But in any case, it is surely in the noise. I can't believe we have any performance metrics accurate or stable enough to detect the change.

@remyhaemmerle-da what do you think?

Reply:
Ah right, I missed the fact that we also allocated before. Then I’m fine with keeping it for now but in the future, let’s please move any unrelated changes to separate PRs.